git-find-ignored-files.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+#
+# git-find-ignored-files.sh
+#
+# Find all files present in the work tree that are ignored by .gitignore
+#
+# Author: Patrick Lühne, https://www.luehne.de/
+#
+
+key="$1"
+
+function print_help
+{
+	echo "usage: git-find-ignored-files [-s | --sort-by-size] [--help]"
+}
+
+if [[ $# -gt 1 ]]
+then
+	print_help
+	exit 1
+fi
+
+case $key in
+	-h|--help)
+		print_help
+		exit 0
+		;;
+	-s|--sort-by-size)
+		find . -type f -print0 | git check-ignore --no-index -z --stdin | xargs -0 du -shc | sort -h
```

Comment:
nit-pick: Do you see a way to make that a bit less redundant? E.g. you could define a variable here and then `eval` the stuff below?

Reply:
Will do!